frontend/src/bundles/employers/components/sorting-dropdown/sorting-dropdown.tsx

```diff
@@ -0,0 +1,42 @@
+import { InputAdornment } from '@mui/material';
```

Comment:
why do we need this component?

Reply:
moved InputAdornment into common Select component as optional props. Since this field has no label, I decided to place this adornment to clarify the purpose of the field. Or we can add 'Sort by ...' to all options, but that, a lot more text and sorting field will be left unlabeled.